src/api/procedures/modifyCaCheckpoint.ts

```diff
@@ -53,11 +57,11 @@ export async function prepareModifyCaCheckpoint(
     if (!exists) {
       throw new PolymeshError({
         code: ErrorCode.ValidationError,
-        message: 'Checkpoint Schedule no longer exists',
+        message: messageNotExist,
```

Comment:
This wasn't supposed to be the same message. It should be "Checkpoint Schedule doesn't exist", not "Checkpoint doesn't exist"